diff -up openconnect-7.07/library.c.orig openconnect-7.07/library.c
--- openconnect-7.07/library.c.orig	2016-07-27 18:14:15.171405176 -0600
+++ openconnect-7.07/library.c	2016-07-27 18:14:22.546399064 -0600
@@ -94,7 +94,7 @@ struct openconnect_info *openconnect_vpn
 #ifdef ENABLE_NLS
 	bindtextdomain("openconnect", LOCALEDIR);
 #endif
-	openconnect_set_protocol(vpninfo, "anyconnect");
+	openconnect_set_protocol(vpninfo, "nc");
 	return vpninfo;
 
 err:
